tasks/sanity-inspect-image.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-inspect-image
+spec:
+  params:
+    - name: SOURCE_IMAGE
+      description: the fully qualified image name
+      default: quay.io/hongweiliu/skopeo-inspect-test:latest
+      type: string
+    - name: BASE
+      description: Inspecting base image for source image
+      default: 'false'
+      type: string
+  steps:
+  - name: inspect-image
+    image: quay.io/redhat-appstudio/hacbs-test:latest
+    script: |
+      IMAGE_INSPECT_FILE=$(workspaces.conftest-ws.path)/image_inspect.json
+      BASE_IMAGE_INSPECT_FILE=$(workspaces.conftest-ws.path)/base_image_inspect.json
+      if [ $(params.BASE) == 'true' ];then
+        echo "Inspecting base image for source image: $(params.SOURCE_IMAGE)"
+        BASE_IMAGE_INSPECT_FILE=$(workspaces.conftest-ws.path)/base_image_inspect.json
+        if [ -f "$BASE_IMAGE_INSPECT_FILE" ];then
+          echo "$BASE_IMAGE_INSPECT_FILE exists"
+        else
+          BASE_IMAGE="$(jq -r ".Labels.\"org.opencontainers.image.base.name\"" $IMAGE_INSPECT_FILE)"
+          skopeo inspect --no-tags docker://$BASE_IMAGE > $BASE_IMAGE_INSPECT_FILE
+        fi
+      else
+        echo "Inspecting source image $(params.SOURCE_IMAGE)"
+        if [ -f "$IMAGE_INSPECT_FILE" ];then
+          echo "$IMAGE_INSPECT_FILE exists"
+        else
+          skopeo inspect --no-tags docker://$(params.SOURCE_IMAGE) > $IMAGE_INSPECT_FILE
```

Comment:
I think we want to run `skopeo inspect --no-tags --raw docker://$(params.SOURCE_IMAGE)` here in addition to this (so both with and without `--raw` option) to get the image config metadata.

Reply:
Yes, it exists in the data returned by `skopeo inspect --no-tags --raw docker://quay.io/mkovarik/spring-petclinic:dc80a7f`.
I would build an oci image by the tasks in this repo and them compare the manifest data the docker and oci

Reply:
I tested the `buildah bud` with our tkn pipeline/tasks file for the same Dockerfile, if the format is oci, there are annotations with `org.opencontainers.image.base.digest` and `org.opencontainers.image.base.name` in the data gotten by `skopeo inspect --raw`, but for docker format image, there are not these info.